src/main/java/hudson/plugins/sonar/SonarBuildWrapper.java

```diff
@@ -168,6 +183,18 @@ public void buildEnvVars(Map<String, String> env) {
         env.put(k, v);
       }
     }
+    
+    @Override
+    public boolean tearDown(AbstractBuild build, BuildListener listener) throws IOException, InterruptedException {
+      UrlSonarAction urlAction = SonarUtils.addUrlActionTo(build);
+      
+      // null result means success so far
+      if (urlAction != null && urlAction.isNew() && build.getResult() == null  && build.getAction(BuildSonarAction.class) == null) {
+          build.addAction(new BuildSonarAction(urlAction.getSonarUrl()));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.internal.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)